slatedb/src/db.rs

```diff
@@ -5095,24 +5096,12 @@ mod tests {
 
         // check that read with durability level remote returns value
         let v = db
-            .get_with_options(
-                &b"foo",
-                &ReadOptions {
-                    durability_filter: DurabilityLevel::Memory,
-                    dirty: false,
```

Comment:
I'm guessing this was dropped since default for bool is already `false` in rust (which always blows my mind)?